StortSpelProjekt/Engine/src/ECS/Components/AudioVoiceComponent.cpp

```diff
@@ -51,6 +58,6 @@ void component::AudioVoiceComponent::Stop(const std::wstring& name)
 		Log::PrintSeverity(Log::Severity::WARNING, "Tried to stop Audio that has not been loaded!");
```

Comment:
Not your code, but would be nice if you could add the name of the sound and a line ending to this print.